common/src/main/java/com/microsoft/identity/common/internal/result/AdalBrokerResultAdapter.java

```diff
@@ -296,8 +298,8 @@ private void setServiceExceptionPropertiesToBundle(@NonNull final Bundle resultB
         }
 
         //INTERACTION_REQUIRED is marked as deprecated
-        if (AuthenticationConstants.OAuth2ErrorCode.INVALID_GRANT.equalsIgnoreCase(serviceException.getErrorCode())
-                || AuthenticationConstants.OAuth2ErrorCode.INTERACTION_REQUIRED.equalsIgnoreCase(serviceException.getErrorCode())
+        if (com.microsoft.identity.common.java.AuthenticationConstants.OAuth2ErrorCode.INVALID_GRANT.equalsIgnoreCase(serviceException.getErrorCode())
```

Comment:
I seperated this out into another file in my pr 😆 , but it's fine. i'll resolve it there once this gets in.

Reply:
(did that partly because i don't like this long prefix string)

Reply:
I saw that.